code-challenges.test.js

```diff
@@ -10,23 +10,109 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
+// Jest Test example:
+//  describe("divThree", () => {
+//     it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not", () => {
+//       const object1 = { number: 15 }
+// // Expected output: "15 is divisible by three"
+//       const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+//       const object3 = { number: -7 }
+// // Expected output: "-7 is not divisible by three"
+//       expect(divThree(object1)).toEqual("15 is divisible by three") 
+//       expect(divThree(object2)).toEqual("0 is divisible by three")
+//       expect(divThree(object3)).toEqual("-7 is not divisible by three")
+//     })
+// })
+
+
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
+
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
+describe("twoMore", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
+    const fibLength2 = 10
+// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(twoMore(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(twoMore(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
+// Test Failed:
+//   ● twoMore › returns an array that length containing the numbers of the Fibonacci sequence
+//     ReferenceError: twoMore is not defined
+
+const fibLength1 = 6
+//Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
 
+// Pseudo code
+
+// Input: a number (greater than 2) 
+// Output: an array that length containing the numbers of the Fibonacci sequence
+// Example: fibLength1 = 6 will look like [1, 1, 2, 3, 5, 8]
+
+// Process
+
+// Declare a function named twoMore
+// Declare an array holding the first two array elements called fib
+// Use a for loop and set the starting index at 2, index less than the parameter number, and i++ for the loop to climb by increments of 1 (for(let i - 2; i < number; i++))
+// Declare an array to hold modified numbers called newFib
+// Set function for each iteration to have the next index equal to the sum of the previous two numbers
+// Idea of logic: newFib is equal to index minus one plus index minus 2( newFib[] = newFib[i-1] + newFib[i -2])
+// Use .push to bring all of the newFib elements into the original fib array
+// Return fib (outside of the for loop function)
+
+//Function:
+
+// const fibLength1 = 6
+// const fibLength2 = 10
+function twoMore(num) {
+  let fib = [1, 1]
```

Comment:
Great! Love the descriptive pseudocode and the plan you implemented. Just out of curiosity, is there a reason why you chose to declare the function in this style?

Reply:
Honestly, I think I was reading the prompt and with out thinking wrote ---> function twoMore <--- because I read my pseudo code saying declare a **function** called **twoMore**. I obviously didn't catch that, but I am glad it still worked without using const as my declaration, hahaha.